src/geom/shape/schema.js

```diff
@@ -105,9 +105,7 @@ function _sortValue(value) {
     value = [ value ];
   }
   // 从大到小排序
-  const sorted = value.sort(function(a, b) {
-    return a < b ? 1 : -1;
-  });
+  const sorted = value.sort(function(a, b) { return a < b ? 1 : -1; });
```

Comment:
here is not changed

`value.sort((a, b) => a < b ? 1 : -1);`

I will modify it myself.